### PR TITLE
week3 homework implemented

### PR DESCRIPTION
**Pull request by 1620prabn**

### CODE REVIEW

File: javascript/javascript2/week3/delayfunction/delayfunction.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Create a function called runAfterDelay. It has two parameters: delay and callback. When called the function should wait delay seconds and then call the provided callback function. Try and call this function with different delays and different callback functions
+function runAfterDelay(delay, callback) {
+  setTimeout(() => {
+    console.log(callback);
```
Comment (florinmaticu) [inline javascript/javascript2/week3/delayfunction/delayfunction.js:4]: Just calling the function is sufficient...
```suggestion
    callback;
```

File: javascript/javascript2/week3/loglocation/location.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <div>
+      <h1>Click the button to get your Geolocation.</h1>
+
+      <button id="btn1">Log Location</button>
+
+      <p id="display"></p>
+    </div>
+
+    <script
+      async
```
Comment (florinmaticu) [inline javascript/javascript2/week3/loglocation/location.html:18]: Can you explain the purpose of `async` and `defer` in the context of calling the Google's API ? What is the alternative when we don't want to do this. I am not a web developer but would like to understand :). Imagine that I am an intern and you are and advanced JavaScript developer and I have to change your code but I don't understand it.

Comment (1620prabn) [reply javascript/javascript2/week3/loglocation/location.html:18]: on my understanding, `async` helps to load the script to load- first and fast, if we have multiple script we can use it. and  `defer` loads in the background and when page is loaded it runs. 
in general:
Scripts with defer always execute when the DOM is ready
async attribute means that a script is completely independent and can run quickly

Comment (florinmaticu) [reply javascript/javascript2/week3/loglocation/location.html:18]: Okay :)

**Comment by 1620prabn** [comment]
Yes, i am working on it, will finish it today, i will msg u when i upload
it .

**Comment by 1620prabn** [comment]
@florinmaticu i have added fast press app

**Pull request closed without merging**